packages/falcon-server-env/src/models/Extension.ts

```diff
@@ -58,6 +60,7 @@ export abstract class Extension {
     this.config = config as ExtensionConfig;
     this.extensionContainer = extensionContainer;
     this.eventEmitter = eventEmitter;
+    this.logger = Logger.getFor(`${this.name}-ext`);
```

Comment:
```suggestion
    this.logger = Logger.getFor(`${this.name}`);
```
I think `-ext` postfix is unecessary here